templates/bundles/DemosPlanDocumentBundle/DemosPlanDocument/elements_admin_import.html.twig

```diff
@@ -11,6 +11,16 @@
         </p>
 
         {% include "@DemosPlanDocument/DemosPlanDocument/elements_admin_import_entry.html.twig" %}
+        {% if hasPermission('feature_auto_switch_element_state') %}
+            <div class="u-mt-0_75">
+                <dp-checkbox
+                    id="r_publish_categories"
+                    checked
+                    :label="{ text: Translator.trans('element.state.autoswitch', { state: Translator.trans('published') }) }"
```

Comment:
I find it confusing to (re-)use the already-existing translation key here. `element.state.autoswitch` carries a distinct meaning which does not apply to the current case. I'd rather suggest to add a new translation key `element.state.publish` or the like. To also differentiate what actually happens, i'd also suggest to chose " Kategorie  sofort veröffentlichen".